build.sbt

```diff
@@ -83,6 +84,19 @@ lazy val rdfProtos = (project in file("rdf-protos"))
     publishArtifact := false,
   )
 
+// Intermediate project that generates the Scala code from the protobuf files
+lazy val rdfProtosJava = (project in file("rdf-protos-java"))
+  .enablePlugins(ProtobufPlugin)
+  .settings(
+    name := "jelly-javameta",
+    libraryDependencies ++= Seq(
+      "com.google.protobuf" % "protobuf-java" % protobufV,
+    ),
+    ProtobufConfig / sourceDirectory := baseDirectory.value / "src" / "main" / "protobuf",
```

Comment:
In the scala version we had two dirs:

- protobuf, for Jelly-JVM-specific protos
- protobuf_shared, for the stuff imported from the submodule

The first one was important because it includes protoc options like the package name and that the result should be in multiple files.

Get rid of the scalapb options, keep the rest.

Split the giant 15kLOC `Rdf` class file into multiple with the `java_multiple_files` setting there.

Reply:
doned. had to do some copy and append to file magic for it to work :P